android/src/main/java/com/swmansion/rnscreens/SearchBarManager.kt

```diff
@@ -77,6 +77,13 @@ class SearchBarManager : ViewGroupManager<SearchBarView>() {
         }
     }
 
+    @ReactProp(name = "shouldClearOnSubmit")
+    fun setShouldClearOnSubmit(view: SearchBarView, shouldClearOnSubmit: Boolean?) {
+        if (shouldClearOnSubmit != null) {
+            view.shouldClearOnSubmit = shouldClearOnSubmit
+        }
```

Comment:
Let's change this into

```suggestion
        view.shouldClearOnSubmit = shouldClearOnSubmit ?: false
```